crates/core/src/kernel/snapshot/mod.rs

```diff
@@ -58,9 +59,8 @@ pub struct Snapshot {
     protocol: Protocol,
     metadata: Metadata,
     schema: StructType,
-    // TODO make this an URL
     /// path of the table root within the object store
```

Comment:
This comment may be out of date, since it refers to the path **within** the object store. I assume `table_url` will now capture the full URL to the table?

Reply:
updated!